module/gridss.nf

```diff
@@ -118,3 +118,69 @@ process run_assembly_GRIDSS {
             ${tumor_bam}
         """
     }
+
+process call_sSV_GRIDSS {
+    container params.docker_image_gridss
+
+    publishDir "${params.workflow_output_dir}/output/",
+        pattern: "${tumor_id}.{vcf,vcf.idx}",
+        mode: "copy",
+        saveAs: {
+            "${output_filename}.${sanitize_string(file(it).getName().replace("${tumor_id}.", ""))}"
+            }
+
+    publishDir "${params.workflow_output_dir}/intermediate/${task.process.replace(':', '/')}",
+        pattern: "${tumor_id}.vcf.gridss.working/*",
+        mode: "copy",
+        saveAs: {
+            "${output_filename}.vcf.gridss.working/${output_filename}.${sanitize_string(file(it).getName().replace("${tumor_id}.", ""))}"
+            }
```

Comment:
**comment**: Saving of these intermediate files should be controlled based on the setting of `save_intermediate_files`